cmd/configure/attributes/container/image.go

```diff
@@ -1,12 +1,14 @@
 package container
 
+// ImageInfo holds container image details.
 type ImageInfo struct {
 	Registry    string `json:"container_image.registry,omitempty"`
 	Repository  string `json:"container_image.repository,omitempty"`
 	Tag         string `json:"container_image.tags,omitempty"`
 	ImageDigest string `json:"container_image.digest,omitempty"`
 }
 
+// ToURI returns the full image URI for the container image.
```

Comment:
close but no champagne

This is a function that can use a simple comment, due to the term `URI` maybe unknow.
So in the comment you can elaborate, like: `ToURI returns a full image resource identifier, example "registry.com/myrepo:tag", constructed from struct's contents.`